.gitignore

```
@@ -0,0 +1,3 @@
+.idea/
+__pycache__/
+logs/
```

Maybe add **.pyc since I see them in the working directory on the server
```suggestion
logs/
**.pyc
```